src/data-loaders/RelationalDataLoader.js

```diff
@@ -0,0 +1,64 @@
+import Sequelize from 'sequelize';
+import get from 'lodash/get'
+import Promise from 'bluebird'
+
+export default class RelationalDataLoader {
+  constructor(config) {
+    this.config = config;
+  }
+
+  async load(req) {
+    try {
+      const requiredKeys = [
+        'dbDialect',
+        'dbUsername',
+        'dbPassword',
+        'dbHost',
+        'dbName',
+      ];
+      if (!this.config.rds) {
+        throw new Error('RDS configuration not passed');
+      }
+      const missingKey = requiredKeys.find((key) => {
+        return !this.config.rds[key];
+      });
+      if (missingKey) {
+        throw new Error(`${missingKey} is required.`);
+      }
+      const config = {
+        uri: `${this.config.rds.dbDialect}://${this.config.rds.dbUsername}:${this.config.rds.dbPassword}@${this.config.rds.dbHost}:${this.config.rds.dbPort}/${this.config.rds.dbName}`,
+        options: {
+          host: process.env.DB_HOST,
+          logging: console.log,
+          dialect: process.env.DB_DIALECT,
+          dialectOptions: {
+            decimalNumbers: true,
+            multipleStatements: true,
+          },
+          define: {
+            underScored: true,
+            timestamps: true,
+          },
+        },
+      };
+
+      const sequelize = new Sequelize(config.uri, config.options);
+      const result = await Promise.mapSeries(
+        req.statements,
+        (statement) => {
+          return sequelize.query(statement);
+        },
+      );
+      
+      const rows = get(result, `[1][1].rows`, null)
```

Comment:
@alichherawalla what "shape" does `result` have? (what are the fields/rows returned)